src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -165,83 +133,6 @@ private static bool IsErrorStatus(SecurityStatusPalErrorCode errorCode)
                 errorCode != SecurityStatusPalErrorCode.Renegotiate;
         }
 
-        /// <summary>
-        /// Set connection buffer size
-        /// </summary>
-        /// <param name="handle">SNI handle</param>
-        /// <param name="bufferSize">Buffer size</param>
-        /// <returns>SNI error code</returns>
-        internal uint SetConnectionBufferSize(SNIHandle handle, uint bufferSize)
-        {
-            handle.SetBufferSize((int)bufferSize);
-            return TdsEnums.SNI_SUCCESS;
-        }
-
-        /// <summary>
-        /// Copies data in SNIPacket to given byte array parameter
-        /// </summary>
-        /// <param name="packet">SNIPacket object containing data packets</param>
-        /// <param name="inBuff">Destination byte array where data packets are copied to</param>
-        /// <param name="dataSize">Length of data packets</param>
-        /// <returns>SNI error status</returns>
```

Comment:
Could you use the xml document over the related function in `TdsParserStateObjectManaged`?

Reply:
I can but is it worthwhile? the function is internal and if you can't work out what it does from looking at the name and args you really shouldn't be looking at the codebase. I like to document important things but most of the comments in this file seem low value to me.

Reply:
Right, because you're working on the driver for a long time. Generally, from the open-source perspective, it's good to enrich documents as much as possible to make it easier for new contributors.

Reply:
Added in state parser.